src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java

```diff
@@ -20,6 +21,25 @@ public NameContainsKeywordsPredicate(List<String> keywords) {
     public boolean test(Person person) {
         if (keywords.isEmpty()) {
             return false;
+        }
+        //     if (!isExactField) {
+        //         if (isExact(person)) {
+        //             isExactField = true;
+        //             return true;
+        //         } else {
+        //             return keywords.stream()
+        //                     .allMatch(keyword ->
+        //                             Arrays.stream(person.getName().fullName.split("\\s+"))
+        //                                     .anyMatch(part -> part.toLowerCase().startsWith(keyword.toLowerCase())));
+        //         }
+        //     } else {
+        //         return false;
+        //     }
+        // if (List.of(person.getName().fullName.trim().split(" ")).equals(keywords)) {
+        //     return true;
+        // }
+        else if (keywords.get(keywords.size() - 1).contains("/")) {
```

Comment:
Good change for user to input the exact full name of the person when adding reminders!